Geometry/MTDNumberingBuilder/test/BuildFile.xml

```diff
@@ -34,3 +34,8 @@
   <use name="rootcling"/>
   <flags EDM_PLUGIN="1"/>
 </library>
+
+<bin file="dummyMain.cpp" name="GeometryMTDNumberingBuilderTestDriver">
+  <use name="FWCore/Utilities"/>
+  <flags TEST_RUNNER_ARGS="/bin/bash Geometry/MTDNumberingBuilder/test runTest.sh"/>
+</bin>
```

Comment:
Here is an example of an alternative without a `dummyMain.cpp` 
https://github.com/cms-sw/cmssw/blob/9da67b42631771c95f10f28ddd1f374d709a4f98/FWCore/Framework/test/BuildFile.xml#L391

Reply:
@makortel ok thanks, I copied the examples from other core geometry packages but if we may avoid one file more why not

Reply:
Yeah, the "other way" is more recent and we have tons of tests using the `TestRunner`, so it is much more probable to find an example with that :)